### DISCUSSION TITLE
posts/%E6%90%AD%E5%BB%BAJekyll-%E9%9D%99%E6%80%81%E5%8D%9A%E5%AE%A2%E7%BD%91%E7%AB%99/

### DISCUSSION THREAD

**Discussion by giscus** [lindocedskes/lindocedskes.github.io#1] [Announcements]
# posts/%E6%90%AD%E5%BB%BAJekyll-%E9%9D%99%E6%80%81%E5%8D%9A%E5%AE%A2%E7%BD%91%E7%AB%99/

搭建静态博客到github 并用vercel加速国内访问

http://127.0.0.1:4000/posts/%E6%90%AD%E5%BB%BAJekyll-%E9%9D%99%E6%80%81%E5%8D%9A%E5%AE%A2%E7%BD%91%E7%AB%99/

**Comment by loneCenturion** [comment]
又打算开始写博客了😎
